packages/gatsby-theme/src/components/VideoEmbed/VideoEmbed.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import VideoEmbed from "."
```

Comment:
Since this is the player only, I think there is no need  for  `@storybook/addon-a11y` but I just want to double-check before :+1: this